sources/ElkArte/Controller/Xml.php

```diff
@@ -757,12 +756,12 @@ public function action_parserorder()
 		// Failed validation, tough to be you
 		else
 		{
-			if (!empty($validation_session))
+			if ($validation_session !== true)
```

Comment:
hmm...
Shouldn't this be 
```
($validation_session === true)
```

Reply:
hmm.... okay, never mind, actually the old one seems pretty broken, because validateSession was returning either `true` or a string, so technically it was always not empty.

Reply:
yup, it had been broken for a bit.